controls/datagrid/columns/overview.md

```diff
@@ -91,10 +63,50 @@ xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
 
 <snippet id='datagrid-club-model' />
 
+**6.** Add the `BoolToValueConverter` to the page's resources:
+
+<snippet id='datagrid-columns-converter' />
+
+# Columns Features
+
+Find below a quick overview of the DataGrid's Columns features.
+
+## Column Headers
+
+The top cell of a column is called Header. Its purpose is to set a caption for the column, which describes the data displayed in it. The .NET MAUI DataGrid provides fully customizable column headers, check [Column Headers]({%slug datagrid-column-header%}) for detailed information on them.
+
+## Columns Cell Templates
+
+The DataGrid provides a set of predefined column types such Text Column, Numerical Column, etc. In case you need to extend the functionality of a column, for example customize the default appearance or add more UI elements, use the exposed templates - `CellContentTemplate` and `CellEditTemplate`. For detailed information on them, go to [Columns Cells Templates]({%slug datagrid-cell-templates%}) topic.
+
+## Column Footers
+
+The DataGrid allows you to display additional information which applies to the columns in a specific row placed at the bottom of the control. This row consists of individual footer cells for each column. Take a look at the [Column Footers]({%slug datagrid-column-footer%}) for detailed information.
+
+## Column Resizing
+
+Columns inside the Telerik .NET MAUI DataGrid are resizable by default. The feature is available only on Desktop - WinUI and MacCatalyst. For more details on it go to [Column Resizing]({%slug datagrid-column-resizing%}) topic.
+
+## Columns Width
+
+The DataGrid provides flexible mechanism for setting columns' width through columns' `SizeMode` and `Width` properties. For more details on this go to [Columns Width]({%slug datagrid-columns-width%}) topic.
```

Comment:
The DataGrid provides a flexible mechanism for setting the columns' width through the columns' `SizeMode` and `Width` properties. For more details, see the [Columns Width]({%slug datagrid-columns-width%}) topic.